docs/pages/debugging/devtools-plugins.mdx

```diff
@@ -176,6 +176,36 @@ export default function App() {
 
 In the terminal, run `npx expo start`, press <kbd>shift</kbd> + <kbd>m</kbd> to open the list of dev tools, and then select the React Query plugin. This will open the plugin's web interface, displaying queries as they are used in your app.
 
+### Redux DevTools
```

Comment:
```suggestion
### Redux
```

Let's just call it Redux since it appears Redux DevTools is the name of the actual tool and this a plugin. This is also the pattern we follow for other libraries on this page.